pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -191,6 +188,25 @@ func CreateOrLoadClusterInfo(clusterdContext *clusterd.Context, context context.
 	return clusterInfo, maxMonID, monMapping, nil
 }
 
+func loadCsiSettings(cephClusterSpec *cephv1.ClusterSpec) cephv1.CSIDriverSpec {
+	settings := cephClusterSpec.CSI
+	if len(settings.ReadAffinity.CrushLocationLabels) == 0 {
+		settings.ReadAffinity.CrushLocationLabels = strings.Split(topology.GetDefaultTopologyLabels(), ",")
+	}
+	// if the csi spec does not specify mount options, apply them from the operator env vars
+	if cephClusterSpec.CSI.CephFS.KernelMountOptions == "" {
```

Comment:
```suggestion
	if cephClusterSpec.CSI.CephFS.KernelMountOptions == "" {
```
what if this is not initialise `cephClusterSpec.CSI.CephFS`? 
we'll nil pointer error right?